rct229/rules/section5/section5rule26.py

```diff
@@ -0,0 +1,274 @@
+                        wwr=20.1,
+                    )["solar_heat_gain_coefficient"]
+                    == table_G34_lookup(
+                        climate_zone,
+                        SCC.EXTERIOR_RESIDENTIAL,
+                        "VERTICAL GLAZING",
+                        wwr=30.1,
+                    )["solar_heat_gain_coefficient"]
+                )
+                and (
+                    table_G34_lookup(
+                        climate_zone,
+                        SCC.EXTERIOR_NON_RESIDENTIAL,
+                        "VERTICAL GLAZING",
+                        wwr=0.1,
+                    )["solar_heat_gain_coefficient"]
+                    == table_G34_lookup(
+                        climate_zone,
+                        SCC.EXTERIOR_NON_RESIDENTIAL,
+                        "VERTICAL GLAZING",
+                        wwr=10.1,
+                    )["solar_heat_gain_coefficient"]
+                    == table_G34_lookup(
+                        climate_zone,
+                        SCC.EXTERIOR_NON_RESIDENTIAL,
+                        "VERTICAL GLAZING",
+                        wwr=20.1,
+                    )["solar_heat_gain_coefficient"]
+                    == table_G34_lookup(
+                        climate_zone,
+                        SCC.EXTERIOR_NON_RESIDENTIAL,
+                        "VERTICAL GLAZING",
+                        wwr=30.1,
+                    )["solar_heat_gain_coefficient"]
+                )
+                and (
+                    table_G34_lookup(
+                        climate_zone,
+                        SCC.EXTERIOR_RESIDENTIAL,
+                        "VERTICAL GLAZING",
+                        wwr=0.1,
+                    )["solar_heat_gain_coefficient"]
+                    == table_G34_lookup(
+                        climate_zone,
+                        SCC.EXTERIOR_NON_RESIDENTIAL,
+                        "VERTICAL GLAZING",
+                        wwr=0.1,
+                    )["solar_heat_gain_coefficient"]
+                )
+            )
+            # get standard code data
+            target_shgc_mix = (
+                table_G34_lookup(
+                    climate_zone,
+                    SCC.EXTERIOR_RESIDENTIAL,
+                    "VERTICAL GLAZING",
+                    wwr=bldg_scc_wwr_ratio[SCC.EXTERIOR_MIXED],
+                )["solar_heat_gain_coefficient"]
+                if bldg_scc_wwr_ratio[SCC.EXTERIOR_MIXED] > 0
+                else 0.0
+            )
+            target_shgc_res = (
+                table_G34_lookup(
+                    climate_zone,
+                    SCC.EXTERIOR_RESIDENTIAL,
+                    "VERTICAL GLAZING",
+                    wwr=bldg_scc_wwr_ratio[SCC.EXTERIOR_RESIDENTIAL],
+                )["solar_heat_gain_coefficient"]
+                if bldg_scc_wwr_ratio[SCC.EXTERIOR_RESIDENTIAL] > 0
+                else 0.0
+            )
+            target_shgc_nores = (
```

Comment:
It doesn't matter much since this variable name does not appear in the report, but I believe we have used `nonres` elsewhere instead of `nores`.